tephra-core/src/main/java/co/cask/tephra/persist/HDFSTransactionLog.java

```diff
@@ -123,6 +130,12 @@ public void append(Entry entry) throws IOException {
     }
 
     @Override
+    public void commitMarker(int count) throws IOException {
+      String key = TxConstants.TransactionLog.NUM_ENTRIES_APPENDED;
+      internalWriter.appendRaw(key.getBytes(), 0, key.getBytes().length, new CommitEntriesCount(count));
```

Comment:
Need to use encoding to get bytes here. Also, save the bytes in a variable so that another conversion is not required to get the length.